app/src/main/res/layout/screen_patient_summary.xml

```diff
@@ -68,19 +68,16 @@
         tools:text="House No. 14, Street No. 10/11, Guru Gobind Singh Nagar, Bathinda, Punjab" />
 
       <TextView
-        android:id="@+id/contactTextView"
+        android:id="@+id/bangladeshNationalIdTextView"
```

Comment:
According to the [designs](https://zpl.io/2yGY0Jn), the Bangladesh national ID should be shown *after* the BP passport, but the current implementation shows it *before* the BP passport. Please fix?

Reply:
Sorry, Fixed it.